### DISCUSSION TITLE
Per-Layer and Dynamic Layer Height on Anycubic Photon M3 Series?

### DISCUSSION THREAD

**Discussion by jmkao** [sn4k3/UVtools#464] [Q&A] [answered]
Hi all, I am looking to pick up a new printer, but have specific need of Per-Layer exposure settings.

Does anyone know if the newer Anycubic Photons, particularly the M3, can handle this? I only see the older Photon S specifically mentioned in the compatibility list.

Thanks!

**Comment by sn4k3** [comment, marked as answer]
It can do some stuff, but i dont recommend.
What you can do per layer with any anycubic machine:

1. Layer position if different from previous, which means you can't have two layers on same position.
2. Lift Height (If not using TSMC)
3. Lift Speed (If not using TSMC)
4. Exposure time

Thats all

If you want full freedom you need gcode.
Look at Uniformation Gkone, fully unlocked with UVtools

**Reply by jmkao** [reply]
great info. thanks!
